content/docs/reference-events.md

```diff
@@ -61,15 +61,15 @@ function onClick(event) {
 }
 ```
 
-> Note:
+> ध्यान दें:
 >
-> If you want to access the event properties in an asynchronous way, you should call `event.persist()` on the event, which will remove the synthetic event from the pool and allow references to the event to be retained by user code.
+> यदि आप asynchronous तरीके से इवेंट के प्रॉपर्टीज का उपयोग करना चाहते हैं, तो आपको इवेंट पर `event.persist()` को कॉल करना चाहिए, जो पूल से सिंथेटिक इवेंट को हटा देगा और यूजर कोड द्वारा इवेंट के रेफरेन्सेस को बनाये रखेगा।
 
-## Supported Events {#supported-events}
+## समर्थित इवेंट्स {#supported-events}
 
-React normalizes events so that they have consistent properties across different browsers.
+React इवेंट्स को सामान्य करता है ताकि उनके विभिन्न ब्राउज़रों में सुसंगत गुण हों।
 
-The event handlers below are triggered by an event in the bubbling phase. To register an event handler for the capture phase, append `Capture` to the event name; for example, instead of using `onClick`, you would use `onClickCapture` to handle the click event in the capture phase.
+नीचे दिए गए घटना संचालकों को bubbling चरण में एक इवेंट द्वारा ट्रिगर किया गया है। कैप्चर चरण के लिए एक ईवेंट हैंडलर रजिस्टर करने के लिए, `Capture` को ईवेंट नाम में जोड़ें, उदाहरण के लिए, `onClick` का उपयोग करने के बजाय, आप कैप्चर/पकड़ना चरण में click इवेंट को संभालने के लिए `onClickCapture` का उपयोग करेंगे।
```

Comment:
event handlers could be left as is in English since they are technical terms (just translate to hindi)

Reply:
revisit paragraph as well, looks too complex

Reply:
revisited and corrected.